src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Versioning/CollectionUpgradeManager.cs

```diff
@@ -23,7 +23,7 @@ public class CollectionUpgradeManager : IUpgradeManager
         /// This integer should be incremented when changing any value in the
         /// UpdateIndexAsync function
         /// </summary>
-        internal const int CollectionSettingsVersion = 1;
+        internal const int CollectionSettingsVersion = 2;
```

Comment:
Not your code but can you update the comment above? There is no UpdateIndexAsync function.